packages/utility-features/json-api/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "@k8slens/typescript/config/base.json",
+  "include": ["**/*.ts"],
+  "compilerOptions": {
+    "moduleResolution": "node"
+  }
```

Comment:
Should we have a different `base.json` for this instead of adhoc-tweak?

Reply:
Maybe, we need this tweak probably always when we are importing esm module to commonjs package 🤔 That said there might be a better way (this feels a bit hacky).